functorch/functorch/_src/aot_autograd.py

```diff
@@ -186,12 +186,16 @@ def create_aot_autograd_function(
     compiled_fw = None
```

Comment:
Doesn't need to be done in this PR, but there's enough state here that we should probably be refactoring it out into a separate object.